src/chat.rs

```diff
@@ -417,7 +417,7 @@ impl ChatId {
             // TODO: For a 1:1 chat this currently triggers `Contact::unblock()` on other devices.
             // Maybe we should unblock the contact locally too, this would also resolve discrepancy
             // with `block()` which also blocks the contact.
-            chat.add_sync_item(context, ChatAction::Unblock).await?;
+            chat.sync(context, SyncAction::Unblock).await.ok();
```

Comment:
I'm constantly forgetting this because in my previous C life i used to logging errors at the lowest level, where they are generated and more details are known, passing the log context (prefix etc.) via TLS. It was not possible to report structured errors upstack, only int error codes (and there were lots of error handling built on these error codes). The downside is that many low-level errors are logged which are actually minor and even expected in systems with lots of hardware (there were ideas how to solve this, but not worked out enough).

But let's clarify our current approach. Afaiu errors should be logged either at the upper level or where they are ignored (one might say, turn into warnings) as here. Right?

Reply:
> The problem here is that error is thrown away without any logging at all.

Yes. My question is only about logging. So, errors should be logged before ignoring them, i.e. before `.ok()`. I guess, before handling them too. Are there other well-known conditions for that?